oak_functions_service/src/lookup.rs

```diff
@@ -84,21 +83,33 @@ impl DataBuilder {
 /// In the future we may replace both the mutex and the hash map with something
 /// like RCU.
 pub struct LookupDataManager {
-    data: RwSpinlock<Arc<Data>>,
+    #[cfg(feature = "std")]
```

Comment:
nit: having these spread throughout the code is a bit difficult to read. Perhaps it would be cleaner to do it with use statements, e.g. something like:

```rust
    #[cfg(feature = "std")]
    use parking_lot::Mutex;
    #[cfg(not(feature = "std"))]
    use spinning_top::Spinlock as Mutex;
```

Reply:
Good idea; I put them in different `mod`-s which should make it even clearer.